src/common/config.cpp

```diff
@@ -55,6 +63,13 @@ void Config::initialize(int argc, char** argv, cli::mode mode, bool validate) {
     seed = get<size_t>("seed");
   }
 
+  #ifdef CUDA_FOUND
+  // set if Tensor Cores should be used
+  useTensorCores = !get<bool>("no-tensor-cores");
```

Comment:
This is more like a question: since we can make a boolean option `true` by default and disable it by `--bool-option false|off|no|0`, maybe we should try to avoid introducing more options like `--no-something`? Double negations are not easy to read.

Reply:
Yeah, I was thinking about this. This is like a diagnostic options, where we specifically disable a rather useful feature. We also have --no-restore and --no-shuffle. Usually you would want in all those cases to have them enabled. Using them seems very explicit.